index.html

```diff
@@ -47,7 +47,240 @@ <h1>Hey There.
+                  </div>
+                </div>
+              </div>
+              <hr />
+            </div>
+    
+            <div class="Abiodunskills">
+              <div class="skill">
+                <div class="Abiodunskillheader">
+                  <ul>
+                    <li class="lang"></li>
+                    <li class="language">Language</li>
+                  </ul>
+                </div>
+                <div class="skilllist">
+                  <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                  </ul>
+                </div>
+              </div>
+    
+              <div class="skill">
+                <div class="Abiodunskillheader">
+                  <ul>
+                    <li class="frame"></li>
+                    <li class="frameworks">Frameworks</li>
+                  </ul>
+                </div>
+                <div class="skilllist">
+                  <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                  </ul>
+                </div>
+              </div>
+    
+              <div class="skill">
+                <div class="Abiodunskillheader">
+                  <ul>
+                    <li class="sk"></li>
+                    <li class="skills">Skills</li>
+                  </ul>
+                </div>
+                <div class="skilllist">
+                  <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </section>
+
+          <section class="about-me2">
+            <div class="contact-me">
+                <p>
+                    I'm always interested in 
+                    hearing about new projects, 
+                    so if you'd like to chat please get in touch.   
+                </p>
+                <form action="https://formspree.io/f/xpzbdbeq" method="post">
+                    <ul>
+                        <li class="input-details">
+                            <input type="text" placeholder="Full name" maxlength="30" required>
+                            <hr>
+                        </li>
+                        <li class="input-details">
+                            <input type="email" placeholder="Email address" required>
+                            <hr>
+                        </li>
+                        <li class="text-area">
+                        <textarea name="Enter-text" id="enter-text" cols="41" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
```

Comment:
- [ ] Please remove this section as it is not needed to complete this milestone. It will be required in the upcoming milestones